sparkle/src/components/Dialog.tsx

```diff
@@ -28,7 +29,10 @@ export function Dialog({
   validateVariant = "primary",
   title,
   isSaving,
+  hasBackground = "none",
```

Comment:
`hasBackground` sounds more like a boolean value, can we update to `backgroundType` instead?